app/web/features/search/FilterDialog.tsx

```diff
@@ -198,7 +225,7 @@ export default function FilterDialog({
                 id="last_active_filter"
                 className={classes.marginBottom}
                 value={lastActiveFilter}
-                onChange={(e) => setLastActiveFilter(e.target.value as number)}
+                onChange={(e) => setLastActiveFilter(parseInt(e.target.value as string))}
```

Comment:
this solved this issue :) [actually was easy to spot, not sure how I didn't realise before that this was the bug]
![image](https://github.com/user-attachments/assets/49cc7fae-3d58-4ec3-8a00-7819598f8bd8)